docs/getting-started/walkthroughs/python_table_api.md

```diff
@@ -142,8 +141,13 @@ t_env.execute("tutorial_job")
 {% endhighlight %}
 
 ## Executing a Flink Python Table API Program
+Firstly, you need to prepare input data in "/tmp/input" file.You can choose the following command line to prepare input data:
```

Comment:
Maybe change "You can choose the following command line to prepare input data" to
"You can choose the following command to prepare the input data"?